assignment-2/src/view/DumbComponents/AnswersList.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
```

Comment:
Folder names should be lower case. Also `components` is somehow redundant, so I would call the package directly: `view/dumb` and `view/smart`. It has a nice ring to it!